bindata/network/ovn-kubernetes/common/001-crd.yaml

```
@@ -74,6 +74,53 @@ spec:
                           description: dnsName is the domain name to allow/deny traffic to. If this is set, cidrSelector must be unset.
                           pattern: ^([A-Za-z0-9-]+\.)*[A-Za-z0-9-]+\.?$
                           type: string
+                        nodeSelector:
+                          description: nodeSelector will allow/deny traffic to the
+                            Kubernetes node IP of selected nodes. If this is, set
```

```suggestion
                            Kubernetes node IP of selected nodes. If this is set,
```
I could care less about a comma positioning usually but in CRD yaml descriptions which often customers/users read with the oc describe better to be accurate. (I know this is missing upstream too, but this will be the customer facing change I guess..)